cmd/terraform-mcp-server/instructions.md

```diff
@@ -16,7 +16,8 @@ The Terraform MCP server provides tools for generating better Terraform code thr
 
 ### Registry Tools (Always Available)
 
-- **Provider Discovery**: `get_latest_provider_version` (if unavailable in code) → `search_providers` → `get_provider_details`
+- **Provider Discovery**: `get_latest_provider_version` (if unavailable in code) → `get_provider_capabilities` → `search_providers` → `get_provider_details`
```

Comment:
nitpick: with provider_doc_id available in get_provider_capabilities it could do directly to get_provider_details if doc found